### ISSUE TITLE
Please add site Pure Love Translations

### ISSUE THREAD

**Issue by TimeLimit67** [dteviot/WebToEpub#1394]
Novel Title: Epiphanies of Rebirth [(43.2-97.2)](https://puretl.com/epiphanies-of-rebirth)

Yep. Before installing the test versions of WebToEpub for Chrome and Firefox downloaded from [here](https://drive.google.com/drive/folders/1B_X2WcsaI_eg9yA-5bHJb8VeTZGKExl8?usp=sharing), I can still extract chapters using Default Parser. After that, error kept haunting me like a vengeful ghost.

### What settings did you use?  What didn't work?
* **URL of first chapter:** https://puretl.com/epiphanies-of-rebirth/c43p2
* **CSS selector for element holding content to put into EPUB:** div#yui_3_17_2_1_1721893173908_104.blog-item-inner-wrapper
* **CSS selector for element holding Title of Chapter:** h1.entry-title.entry-title--large.p-name
* **CSS selector for element(s) to remove:**

Again, I don't know what went wrong. I tried different CSS selectors I can find containing the chapter content, but I just can't get through it no matter what I do.

This is the error that always occurred whenever I try to click Test button:
![Screenshot (23)](https://github.com/user-attachments/assets/8acff935-5129-4c9e-a978-8e4a3c2f1433)

**Comment by dteviot** [comment]
@TimeLimit67 

I'm going to have to write a custom parser.  (Wanted content is in the 2nd div.sqs-html-content element.)

This might sort of work in the meantime.  (Lots of garbage)

**CSS selector for element holding content to put into EPUB:** div.blog-item-content
**CSS selector for element holding Title of Chapter:** h1.entry-title
CSS selector for element(s) to remove:  leave blank

For my notes: 14 minutes.

**Comment by dteviot** [comment]
@TimeLimit67 

Test versions for Firefox and Chrome have been uploaded to https://drive.google.com/drive/folders/1B_X2WcsaI_eg9yA-5bHJb8VeTZGKExl8?usp=sharing. Pick the one suitable for you, follow the "How to install from Source (for people who are not developers)" instructions at https://github.com/dteviot/WebToEpub/tree/ExperimentalTabMode#user-content-how-to-install-from-source-for-people-who-are-not-developers and let me know how it goes.
Tested with:
* https://puretl.com/epiphanies-of-rebirth, chapters 1 and 2.  (Chapter titles in ToC are missing some text, but that's because the page is weird.  Not all title text is in the link.)
* https://puretl.com/my-koi, chapters 1 and 2 (ToC layout is different.  Fails to get Title of Story.  Won't fix.)
* https://puretl.com/village-belle, chapters 1 and 2.  (ToC layout is different again)

In short, this site is a pain.

For my notes: 30 minutes work (extra)  44 minutes total.

**Comment by dteviot** [comment]
Updated version (0.0.0.167) has been submitted to Firefox and Chrome stores.
Firefox version is available now.
Chrome might be available in a few hours to 21 days.
